scripts/optimizeImage.sh

```diff
@@ -5,7 +5,7 @@ OUTPUT="$2"
 
 cp "$INPUT" "$OUTPUT" || exit 1
 
-mogrify -format jpg "$OUTPUT" || exit 2
+mogrify -format jxl -strip "$OUTPUT" || exit 2
```

Comment:
Could lines 8 and 9 be combined?

Reply:
It's prolly a better idea to keep the conversion and resizing operations separate for now.